README.md

```diff
@@ -140,27 +173,31 @@ On Linode running as separate VMs with automated server configuration:
 
 If it makes sense we might move cuttlefish and morph.io to AWS as well.
 
-## Requirements
+## <a name='Requirements'></a>Requirements
 
-### Install Vagrant and Capistrano
-For starting local VMs for testing you will need [Vagrant](https://www.vagrantup.com/).
-For deploying code you'll need [capistrano](http://capistranorb.com/)
-You'll need Python 2.6 or 2.7 and virtualenv installed by your OS package manager.
+### <a name='Prerequisites'></a>Prerequisites
+- For starting local VMs for testing you will need [Vagrant](https://www.vagrantup.com/) and a supported provider - our instructions assume [VirtualBox](https://developer.hashicorp.com/vagrant/docs/providers/virtualbox).
+- In order to run Ansible, you'll need Python < 3.12 installed
+	- 3.12 dropped some deprecated language features which cause [Ansible 2.9 and 2.10 to no longer work](https://github.com/ansible/ansible/issues/81946).
+- In order to run Capistrano, you'll need a version of Ruby installed; even better, install [rbenv](https://rbenv.org/) so that you're able to manage multiple versions of Ruby.
+- For deploying code onto dev/test/prod machines, you'll need [capistrano](http://capistranorb.com/)
+- For a few things, including PlanningAlerts deployments, you'll need [Terraform](https://developer.hashicorp.com/terraform/install)
```

Comment:
Again. Not strictly true because everday deployment is still done via capistrano

Reply:
Updated